tests/core.js

```diff
@@ -33,10 +33,10 @@ require([
             util.init({
                 core: {
                     onReady: function () {
-                        var dup;
                         util.addQuestion("Text", "question1");
-                        dup = util.addQuestion("Text", "question2");
-                        dup.p.nodeID = "question1";
+                        util.addQuestion("Text", "question2");
+                        var target = $("[name='property-nodeID']");
+                        target.val('question1').change();
```

Comment:
I found when writing my other tests that by modifying the mug directly doesn't go through the set method, which can have side effects. Went ahead and changed this one, but there may be others.